src/main/java/edu/ucsb/cs156/team02/repositories/UCSBSubjectRepository.java

```diff
@@ -0,0 +1,12 @@
+package edu.ucsb.cs156.team02.repositories;
+
+import edu.ucsb.cs156.team02.entities.UCSBSubject;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface UCSBSubjectRepository extends CrudRepository<UCSBSubject, String> {
+  Iterable<UCSBSubject> findBySubject(String subject);
+}
```

Comment:
See the Slack thread [here](https://ucsb-cs156-w22.slack.com/archives/C030KT692A3/p1643766836800789). Because there is no member in `UCSBSubject` called `subject` the method `findBySubject` will — uh, I don't know what it will do. Probably `RuntimeException`.

Reply:
Thanks, just saw it. Resolved.